sklearn/mixture/tests/test_gaussian_mixture.py

```diff
@@ -127,56 +128,39 @@ def test_gaussian_mixture_attributes():
 
     n_components_bad = 0
     gmm = GaussianMixture(n_components=n_components_bad)
-    msg = (
-        f"Invalid value for 'n_components': {n_components_bad} "
-        "Estimation requires at least one component"
-    )
-    with pytest.raises(ValueError, match=msg):
+
+    with pytest.raises(ValueError):
```

Comment:
I'd rather keep checking the error message, here and below as well. It might have change after switching to use `check_scalar` in the parameter checks.

Reply:
@ariosramirez can you check this suggestion? Thanks!